protoc-gen-kotlin/lib/src/commonMain/kotlin/pbandk/gen/CodeGenerator.kt

```diff
@@ -13,6 +16,7 @@ open class CodeGenerator(
         file.kotlinPackageName?.let { line("package $it") }
         file.types.forEach { writeType(it) }
         file.types.filterIsInstance<File.Type.Message>().forEach { writeMessageExtensions(it) }
+        file.extensions.forEach { writeExtension(it) }
```

Comment:
Move this up a line so that it's before the `writeMessageExtensions()` line. We try to keep the interesting bits of the generated code (i.e. the type definitions) at the top, and the uninteresting implementation details (i.e. Kotlin extension methods) at the bottom.

Reply:
This has been moved up.